gapis/api/templates/vulkan_gfx_api_extras.tmpl

```diff
@@ -101,6 +101,7 @@ bool Vulkan::replayCreateVkInstanceImpl(Stack* stack,
   std::vector<const char*> layers(
       pCreateInfo->ppEnabledLayerNames,
       pCreateInfo->ppEnabledLayerNames + pCreateInfo->enabledLayerCount);
+  layers.push_back("APITiming");
```

Comment:
Is this in the right place, shouldn't it be below where we add the VirtualSwapchain layer?

Reply:
This were unintentionally added. I don't want them inserted at all times, will remove.